src/state/persisted/schema.ts

```diff
@@ -0,0 +1,59 @@
+import {deviceLocales} from '#/platform/detection'
+
+// only data needed for rendering account page
+type Account = {
+  service: string
+  did: `did:plc:${string}`
+  refreshJwt: string
+  accessJwt: string
+  handle: string
+  displayName: string
+  aviUrl: string
+}
+
+export type Schema = {
+  colorMode: 'system' | 'light' | 'dark'
+  accounts: Account[]
+  currentAccount: Account | undefined
+  lastEmailConfirmReminder: string | undefined
+
+  // preferences
+  primaryLanguage: string // should move to server
+  contentLanguages: string[] // should move to server
+  postLanguage: string // should move to server
+  postLanguageHistory: string[] // should move to server
+  requireAltTextEnabled: boolean // should move to server
+  mutedThreads: string[] // should move to server
+
+  // should move to server?
+  invitedUsers: {
+    seenDids: string[]
+    copiedInvites: string[]
+  }
+
+  onboarding: {
+    step: string
+  }
```

Comment:
The other properties felt OK to flatten, but these two I decided to leave as objects in case we need to expand their child props.